server/src/main/scala/com/cloudera/livy/server/ApiVersioningSupport.scala

```diff
@@ -0,0 +1,98 @@
+package com.cloudera.livy.server
+
+import javax.servlet.http.HttpServletRequest
+
+import org.scalatra.{NotAcceptable, ScalatraBase}
+
+/**
+  * Livy's servlets can mix-in this trait to get API version support.
```

Comment:
I'm using Intellij. That's the default comment syntax it generates.
I fixed it manually but I've some concerns on a coding style that's so strict that they are incompatible with popular IDEs.